stix2/v20/common.py

```diff
@@ -145,7 +145,14 @@ class _Custom(cls, _STIXBase):
 
             def __init__(self, **kwargs):
                 _STIXBase.__init__(self, **kwargs)
-                cls.__init__(self, **kwargs)
+                try:
+                    cls.__init__(self, **kwargs)
+                except (AttributeError, TypeError) as e:
+                    # Don't accidentally catch errors raised in a custom __init__()
+                    if ("has no attribute '__init__'" in str(e) or
+                            str(e) == "object.__init__() takes no parameters"):
+                        return
+                    raise e
```

Comment:
Codecov is failing because this line isn't tested because none of our tests have a `CustomMarking` with a custom `__init__` function that raises an exception like our `NewType` custom object does in `test_custom.py`. Maybe we should change https://github.com/oasis-open/cti-python-stix2/blob/4990b7edb2e2b9d0377c4fe8ee8d47063f1e5313/stix2/test/test_markings.py#L190 and add a test with `pytest.raises`.

Reply:
That line is also not covered for CustomObject (potentially any of the other Custom decorators). https://codecov.io/gh/oasis-open/cti-python-stix2/src/ef3ce9f6f0a678e604bda63f9eb5742ed53f8eff/stix2/v20/sdo.py#L359

Reply:
Oh, because our tests' CustomObject's `__init__` raises a `ValueError`, which is not caught here.